content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -6,20 +6,56 @@ weight: 300
 toc: true
 ---
 
-You can modify the engine settings registry file for **Asset Processor** in `Registry/AssetProcessorPlatformConfig.setreg` to perform the following tasks:
+Many of the rules for how **Asset Processor** will process content for your project are controlled by settings registry values. You can view the default values in [`Registry/AssetProcessorPlatformConfig.setreg`](https://github.com/o3de/o3de/blob/development/Registry/AssetProcessorPlatformConfig.setreg). This file is well commented, and full of examples you can use for specific configuration on your project's asset pipeline.
 
-* Add new source asset types for Asset Processor.
-* Update existing file type rules.
-* Update the ignore list.
-* Add or remove target platforms.
+To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to set up and deploy to additional platforms.
 
-    {{< note >}}
-Asset Processor automatically builds assets for the host platform. For example, if Asset Processor is running on Windows, Asset Processor builds Windows assets even if pc is not enabled in the `.setreg` file.
-    {{< /note >}}
+By default, the Asset Processor will create a worker job for each logical core on the host machine. If you wish to manage this yourself, you can adjust the setting `Amazon/AssetProcessor/Settings/Jobs/maxJobs`. A value of zero will have the Asset Processor create a job for each logical core on your machine. A value of one can be useful when debugging, if you wish to attach a debugger to a running builder processing a specific asset. Other values lower than your logical core count can be useful to manage the impact asset processing has on your machine while you are trying to perform other tasks, with the tradeoff of a longer time before all assets are processed.
 
-* Add scan directories for Asset Processor.
+The asset cache server's address can be set using the setting path `Amazon/AssetProcessor/Settings/Server/CacheServerAddress`. In most cases this will be a network share, an example for Windows looks like `T:\\AssetServerCache`. Documentation for this feature is still in progress, feel free to reach out on Discord if you have questions about the Asset Cache Server in the meantime.

```

Comment:
Saw the previous discussion on this, one small suggestion.

```suggestion
The Asset Cache server's address can be set using the setting path `Amazon/AssetProcessor/Settings/Server/CacheServerAddress`. In most cases this will be a network share, an example for Windows looks like `T:\\AssetServerCache`. This feature is in development. If you have questions about the Asset Cache server, reach out us on the [O3DE Discord](https://discord.com/invite/o3de).
```